vignettes/ChromBand.Rmd

```diff
@@ -0,0 +1,578 @@
+output:
+  BiocStyle::html_document:
+    toc: false
+    theme: journal
+bibliography: 
+    - cat.bib
+link-citations: true
+---
+
+# Chromosome bands
+
+Typically, in higher organisms, each chromosome has a centromere and two arms. The short arm is called the p arm and the longer arm the q arm. Chromosome bands (see Figure [1](#fig:chr12ideogram){reference-type="ref" reference="fig:chr12ideogram"}) are identified by differential staining, usually with Giemsa-based stains, and many disease-related defects have been mapped to these bands; such mappings have played an important role in classical cytogenetics. With the availability of complete sequences for several genomes, there have been efforts to link these bands with specific sequence locations [@FureyHaussler]. The estimated location of the bands in the reference genomes can be obtained from the UCSC genome browser, and data linking genes to particular bands can be obtained from a variety of sources such as the NCBI. This vignette demonstrates tools that allow the use of categories derived from chromosome bands, that is, the relevant categories are determined *a priori* by a mapping of genes to chromosome bands.
+
+Figure [1](#fig:chr12ideogram){reference-type="ref" reference="fig:chr12ideogram"} shows an ideogram of human chromosome 12, with the band 12q21 shaded. As shown in the figure, 12q21 can be divided into more granular levels 12q21.1, 12q21.2, and 12q21.3. 12q21.3 can itself be divided at an even finer level of resolution into 12q21.31, 12q21.32, and 12q21.33. Moving towards less granular bands, 12q21 is a part of 12q2 which is again a part of 12q. We take advantage of this nested structure of the bands in our analysis.
+
+```{r,echo=FALSE, eval=FALSE}
+if (!requireNamespace("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+BiocManager::install("Category")
+```
+
+::: Schunk
+::: Sinput
+\> library("Category")\
+\> library("ALL")\
+\> library("hgu95av2.db")\
+\> library("annotate")\
+\> library("genefilter")\
+\> ##library("SNPchip")\
+\> library("karyoploteR")\
+\> library("geneplotter")\
+\> library("limma")\
+\> library("lattice")\
+\> library("graph")\
+:::
+:::
+
+# Data Preparation
+
+For illustration, we use a microarray dataset [@Chiaretti2005] from a clinical trial in acute lymphoblastic leukemia (ALL). The data are described in Chapter 2 of [@BiocCaseStudies]. For the
+
+<figure id="fig:chr12ideogram">
+
+::: center
+```{r chr12ideogram, echo=FALSE, fig.align="center", out.width="100%", out.height="100%"}
+knitr::include_graphics("figures/chr12ideogram.png")
+```
+:::
+
+<figcaption>Ideogram for human chromosome 12. The p arm is on the left, the q arm is on the right, and the centromere is indicated by a notch. The shaded bands together represent 12q21. This band is composed of three sub-bands: 12q21.1, 12q21.2, and 12q21.3. The last of these is composed of sub-sub-bands 12q21.31, 12q21.32, and 12q21.33.</figcaption>
+
+</figure>
```

Comment:
Replace 56-66 with the code block to generate the figure. Create a code block containing the code in https://github.com/Bioconductor/Category/pull/4/files#diff-fe345a9843884e8331a5988cdd3d46c9275d714c6e87261a3822aa3f63aba921L122-L136. Keep `chr12ideogram` as the name. To display the code properly, you'll have to provide options: `fig.width`, `fig.height`, `fig.align`, and set the text in line 64 as `fig.cap`.

Reply:
Ok, thank you very much for your wise guided instructions, I am on it and fixing them immediately . I am grateful.